src/data/init/init-moves.ts

```diff
@@ -1133,7 +1143,7 @@ export function initMoves() {
         i18next.t("moveTriggers:isTighteningFocus", { pokemonName: getPokemonNameWithAffix(user) }),
       )
       .punchingMove()
-      .condition((user, _target, _move) => !user.turnData.attacksReceived.find((r) => r.damage)),
+      .condition(new FocusPunchCondition),
```

Comment:
```suggestion
      .condition(new FocusPunchCondition()),
```
Wtf, why doesn't this produce an error?